README.md

```diff
@@ -0,0 +1,32 @@
+# ftrack OpenAssetIO Plugin
+
+A skeleton OpenAssetIO ManagerPlugin that binds ftrack to the API
+
+> **Warning**
+> This plugin is highly experimental, and subject to significant
+> change.
+
+## Getting started
+
+Place the `plugin` on `$OPENASSETIO_PLUGIN_PATH`. It requires
+`openassetio` to be available to python at runtime.
+
+The plugin provides a manager with the identifier `com.ftrack`.
+
+## Testing
+
+```bash
+python -m venv .venv
```

Comment:
[consider] Arguably obvious, but I think we should be explicit which directory you should be in when running command sequences

Reply:
:+1: